synapse/lib/cell.py

```diff
@@ -575,7 +583,7 @@ def addHttpApi(self, path, ctor, info):
             logger.error(f'Failed to listen on unix socket at: [{sockpath}][{e}]')
             logger.error('LOCAL UNIX SOCKET WILL BE UNAVAILABLE')
         except Exception as e:  # pragma: no cover
-            logging.exception('Unknown dmon listen error.')
```

Comment:
why is `{e}` used here? logging.Exception already captures the traceback.

Reply:
undone.